src/core/BanksUser.ts

```diff
@@ -86,6 +86,45 @@ export class BanksUser {
     return new BanksUser(banksUser, requestBuilder);
   }
 
+  /**
+   * Fetch all Accounts related to a BankUserID
+   *
+   * @param id Id of the BanksUser to fetch
+   * @param requestBuilder Service account request builder
+   *
+   * @returns Array of BanksUserAccount
+   */
+  public static async getAccountsByID(id: string, requestBuilder: RequestBuilder): Promise<BanksUserAccount[]> {
```

Comment:
If you add the `ById` suffix, it means that you are looking for a specific account

```suggestion
  public static async getAccounts(id: string, requestBuilder: RequestBuilder): Promise<BanksUserAccount[]> {
```